drivers/i2c/i2c_handlers.c

```diff
@@ -4,15 +4,37 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <syscall_handler.h>
 #include <i2c.h>
+#include <string.h>
+#include <syscall_handler.h>
 
 _SYSCALL_HANDLER(i2c_configure, dev, dev_config)
 {
 	_SYSCALL_OBJ(dev, K_OBJ_DRIVER_I2C);
 	return _impl_i2c_configure((struct device *)dev, dev_config);
 }
 
+static u32_t copy_msgs_and_transfer(struct device *dev,
+				    const struct i2c_msgs *msgs,
+				    u8_t num_msgs,
+				    u16_t addr)
+{
+	struct i2c_msgs copy[num_msgs];
```

Comment:
the data type is "struct i2c_msg"

try building tests/drivers/build_all with test_build_sensors_a_m.conf on qemu_x86 to see the build error